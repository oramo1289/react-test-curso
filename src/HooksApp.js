import React from 'react'

export const HooksApp = () => {
    return (
        <div>
            Hola Mundo
        </div>
    )
}
