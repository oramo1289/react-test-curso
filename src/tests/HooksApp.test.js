import React from 'react';
import { shallow } from 'enzyme';
import { HooksApp } from '../HooksApp';

describe('pruebas en hooksapp component', () => {
    const wrapper = shallow( <HooksApp/> );

    test('should match snapshot', () => {
        expect(wrapper).toMatchSnapshot();
    })
    
})