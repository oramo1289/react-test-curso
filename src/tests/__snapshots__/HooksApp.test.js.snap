// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pruebas en hooksapp component should match snapshot 1`] = `
<div>
  Hola Mundo
</div>
`;
